### DISCUSSION TITLE
When can we expect Nuxt 3 to release for production?

### DISCUSSION THREAD

**Discussion by aligajani** [nuxt/nuxt#18049] [General]
Any timelines would be nice.

**Comment by sjonibek** [comment]
Who knows answer to this question? I'm interesting too

**Comment by misaon** [comment]
I assume around end of 1Q 2022

**Comment by fellz** [comment]
Interested too, our company wants to start project on nuxt3 ( with production) and it would be nice to hear from founders about plans for release
@danielroe

**Comment by paulvonber** [comment]
Let's be fair, nuxt team doing great job, but lot of things still need to be polished also the whole ecosystem needs to be adopted as well, personally I don't think that it could be ready earlier than Q3 2022, but let's keep fingers crossed 😵‍💫

**Comment by ghonchesefidi** [comment]
Our company is waiting to start a new project with Nuxt3. Do you think, should we still hold the project or start it with Nuxt 2? What is your advice?

**Comment by HendrikJan** [comment]
Part of an answer can be found here: nuxt/nuxt#17918

**Comment by vencho-mdp** [comment]
Any updates on this?
